packages/bpk-mixins/src/mixins/_svgs.scss

```diff
@@ -42,9 +42,7 @@
     $err: (
       'Could not find "' +
         $icon +
-        '" icon. Refer to ' +
-        'https://backpack.github.io/components/atoms/icons' +
-        ' when specifying icon names.'
+        '" icon. Refer to Refer to our documentation for the list of supported icons.'
```

Comment:
why not include the link here anymore?

Reply:
Ah right, this was the first repo I updated and was thinking to keep specific links like this one to a minimum as it increases the risk of having outdated links in case we change the page name or something like that.
I've also realised I have a typo 'Refer to Refer to' 😄